app/views/map/_indoor.html.erb

```diff
@@ -3,13 +3,33 @@
         "fillOpacity": 0.0,
         "color": "#444"
     };
+    let roomLayer;
+    let roomTooltip;
     <% @buildings.each do |building|%>
         <% building.rooms.each do |room|%>
+
+            roomLayer = L.geoJSON(<%= raw(JSON.generate(room.to_geojson)) %>, 
+            {style: indoorStyle});
+            roomTooltip = L.tooltip({
```

Comment:
this object can be extracted. As you can see in the source code, this object is duplicated 
![image](https://user-images.githubusercontent.com/62988232/148639115-b8d23487-47b1-408a-8ff0-184b20378544.png)

Reply:
Are you referring to the full name of the room? So you propose to first store the Ruby variable `room.full_name` in a JavaScript variable and then use the JS variable from then on? We could definitely do that 👍